common/djangoapps/student/views.py

```diff
@@ -1612,6 +1612,7 @@ def _do_create_account(form, custom_form=None):
     ]
     profile = UserProfile(
         user=user,
+        usa_region=form.data['usa_states'] if 'usa_states' in form.data.keys() else None,
```

Comment:
why don't you use `form.cleaned_data`?

Reply:
Because field `Regions` is dynamic(show if select  USA value into City field) and form don`t know anything about the Region field.
I send value through `form.data` because it`s the only way send region data to back-en

Reply:
but into profile you always save it, maybe there is sense to add this field to the form...

Reply:
`form.data.get('usa_states')` will work